src/dialog/src/Dialog.js

```diff
@@ -14,7 +14,7 @@ const animationEasing = {
 
 const ANIMATION_DURATION = 200
 
-const openAnimation = css.keyframes('openAnimation', {
+const openAnimation = keyframes({
```

Comment:
Assuming these two calls are equivalent?

Reply:
Roughly, minus the first arg (optional name) which was removed in this PR.